docs/howto/craft-for-platforms/example-compiled-app.rst

```diff
@@ -0,0 +1,69 @@
+.. _example-compiled-app:
+
+Example compiled app
```

Comment:
From the title, I was expecting a page on C, C++, go, rust, etc platforms.

You use "pre-compiled" in the sentence below and it makes more sense. 

Going further, I like "pre-built" even better because it encapsulates apps using non-compiled languages and aligns terminology with Snapcraft's "build" step.

Reply:
Noted, I'll go with "pre-built".

Reply:
2b59176